### PR TITLE
Add a script to close bugs for EOLed releases

### PR DESCRIPTION
**Pull request by kshlm**

### CODE REVIEW

**Comment by nigelbabu** [comment]
Might want to quickly run it through shellcheck.net

File: close-bugs-EOL.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    exit 1
+  fi
+}
+
+close_bugs()
+{
+	COMMENT="This bug is getting closed because GlusteFS-${VERSION} has reached its end-of-life.
+
+Note: This bug is being closed using a script. No verification has been performed to check if it still exists on newer releases of GlusterFS.
+If this bug still exists in newer GlusterFS releases, please reopen this bug against the newer release.
```
Comment (nixpanic) [inline close-bugs-EOL.sh]: Not everyone can "reopen" bugs. It is better to ask for reporting a new bug against a maintained version.

Comment (nixpanic) [reply close-bugs-EOL.sh]: Probably point yo https://www.gluster.org/community/release-schedule/ too ;-)

File: close-bugs-EOL.sh [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  if [ $? -ne 0 ]; then
+    echo "`bugzilla` command is missing"
+    echo "Install `python-bugzilla` before running this script again"
+    exit 1
+  fi
+}
+
+close_bugs()
+{
+	COMMENT="This bug is getting closed because GlusteFS-${VERSION} has reached its end-of-life.
```
Comment (nixpanic) [inline close-bugs-EOL.sh]: Also, +r in GlusteFS

**Review by raghavendra-talur — DISMISSED** [review]

**Review by raghavendra-talur — COMMENTED** [review]
LGTM after Niels' comments are addressed.

**Comment by kshlm** [comment]
Closing this old PR. Gluster.org isn't using bugzilla anymore.

**Pull request closed without merging**